doc/querying-prometheus.md

```diff
@@ -0,0 +1,190 @@
+# Querying Prometheus using the Google Cloud Console
+
+Metrics data from our GKE clusters is persisted in [GCP's Managed Prometheus][managed-prom] service.
+For the time being, the main entry point for querying this data is GCP's "Metric Explorer", which lets you run queries against the Prometheus instance.
+
+[managed-prom]: https://cloud.google.com/stackdriver/docs/managed-prometheus
+
+<!-- MarkdownTOC -->
+
+- [Introduction to basic concepts used in Prometheus](#introduction-to-basic-concepts-used-in-prometheus)
+    - [Labeled timeseries data](#labeled-timeseries-data)
+    - [Relevant metric types](#relevant-metric-types)
+    - [Prometheus Query Language](#prometheus-query-language)
+- [Getting relevant KPIs for Wikibase.cloud](#getting-relevant-kpis-for-wikibasecloud)
+    - [ElasticSearch](#elasticsearch)
+    - [Istio Service Mesh](#istio-service-mesh)
+    - [Nginx Ingress Controller](#nginx-ingress-controller)
+
+<!-- /MarkdownTOC -->
+
+## Introduction to basic concepts used in Prometheus
+
+[Prometheus][prom] is a database ecosystem designed specifically for storing and querying timeseries data emitted by monitoring systems.
+
+[prom]: https://prometheus.io/
+
+### Labeled timeseries data
+
+Each datapoint in a time series is a timestamped tuple of a metric name and a numeric (float64) value.
+In addition to that, labels (arbitrary key value pairs) can be applied to a datapoint.
+
+For example, a HTTP server that has since start served 2.319 GET requests with a 200 status code, could emit data like this: 
```

Comment:
:de: detected :P (I know I know, it hurt's my brain too)

```suggestion
For example, a HTTP server that has since start served 2,319 GET requests with a 200 status code, could emit data like this: 
```

Reply:
Ugh.